guice-intro/src/main/java/com/baeldung/examples/guice/marker/Communicator.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.examples.guice.marker;
+
+/**
+ *
+ * @author Baeldung
+ */
+public interface Communicator {
+    

```

Comment:
That's fine, I thought it might be just that, but wanted to check with you to make sure.